src/components/Feedback/FileProgress/CompactFileProgress.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC, useEffect, useMemo, useState } from 'react';
+import { FileWithPath } from 'react-dropzone';
+
+import { CircularProgress, Icon } from '@equinor/eds-core-react';
+import { clear, error_outlined } from '@equinor/eds-icons';
+import { tokens } from '@equinor/eds-tokens';
+
+import { readUploadedFileAsText } from '../../Navigation/TopBar/Help/Feedback/Feedback.utils';
+import {
+  AdditionalText,
+  CloseButton,
+  CompactFileProgressContainer,
+  FileTooltip,
+  ImageWrapper,
+  LoadingWrapper,
+  Rejection,
+} from './CompactFileProgress.styles';
+import {
+  CompactFileProgressBaseProps,
+  FileProgressPropsExtension,
+} from './FileProgress';
+
+const { colors } = tokens;
+
+interface CompactFileProgressProps
+  extends CompactFileProgressBaseProps,
+    FileProgressPropsExtension {}
+
+const CompactFileProgress: FC<CompactFileProgressProps> = ({
+  file,
+  progressPercent,
+  isError,
+  shortErrorText,
+  showCompleteState,
+  fullErrorText,
+  handleOnClick,
+}) => {
+  const [src, setSrc] = useState('');
+
+  useEffect(() => {
+    const getSrc = async (file: FileWithPath) => {
```

Comment:
This useEffect can be replaced with a useMemo like this:
```
const src = useMemo(() => ..., []);
```

Reply:
The reading of the file is async, can i do that in a useMemo?

Reply:
Hmm, no I don't think so. Missed it being async 😅  You can keep it as is 💯